src/neo4j-arc/rollup.config.js

```diff
@@ -13,42 +11,34 @@ const importsWithPaths = [
 ]
 
 const dependenciesNotToBundle = Object.keys({
```

Comment:
rename since not bundling anymore perhaps?

Reply:
Agreed! Eager to merge to get the tests through, so I'll add a note to fix this later!